doc/sphinxman/source/chemps2.rst

```diff
@@ -85,7 +85,7 @@ Installation
   and rebuild |PSIfour| to detect CheMPS2 and activate dependent code.
 
 * Previous bullet had details. To build |PSIfour| from source and use
```

Comment:
Either remove the bullet point or add a new link.

Reply:
yes, these were all referring to a nonexistant condapsi4dev docs section. I was planning on ditching all this repeat documentation, so the incomplete bullet wouldn't have lasted, but removing it is better for now.